.ci/docker/ubuntu/Dockerfile

```diff
@@ -188,7 +190,8 @@ ARG BUILD_ENVIRONMENT
 ENV BUILD_ENVIRONMENT ${BUILD_ENVIRONMENT}
 
 # Install LLVM dev version (Defined in the pytorch/builder github repository)
-COPY --from=pytorch/llvm:9.0.1 /opt/llvm /opt/llvm
+# snadampal: disabling it as this layer doesn't exist for aarch64 linux, need to make it conditional
```

Comment:
Can we update this PR to make this conditional ?

Reply:
yes, looks like there is no conditional COPY in Dockerfile. I'm checking if there any workaround.

Reply:
I think better option is to have the `pytorch/llvm:9.0.1` available for aarch64-linux as well.

Reply:
I have work aounded the missing llvm source build layer COPY on aarch64-linux.